src/EditorFeatures/CSharp/Formatting/CSharpFormattingInteractionService.cs

```diff
@@ -92,7 +92,7 @@ public bool SupportsFormattingOnTypedCharacter(Document document, char ch)
             CancellationToken cancellationToken)
         {
             var parsedDocument = ParsedDocument.CreateSynchronously(document, cancellationToken);
-            var options = textBuffer.GetSyntaxFormattingOptions(_editorOptionsService, parsedDocument.ProjectServices, explicitFormat: true);
+            var options = textBuffer.GetSyntaxFormattingOptions(_editorOptionsService, parsedDocument.LanguageServices, explicitFormat: true);
```

Comment:
`ProjectServices` was renamed to `LanugageServices` on behalf of the API committee.  This was preferred as the type exposes ILanguageServices.  I originally didn't name it that way as we we had HostLanguageServices and i was naming the type `HostXXX`.  However, the API group decided to drop `Host`.  So now we have:

`HostWorkspaceServices` (existing, mutable) and its immutable counterpart: `SolutionServices`
`HostLanguageServices` (existing, mutable) and its immutable counterpart: `LanguageServices`